Panic when listing notifiers

```
scalingo -a <appname> notifiers
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/Scalingo/cli/notifiers.eventTypesToString(0xc000133000, 0x23, 0x2a, 0x1095bb0, 0x0, 0x0, 0x0, 0x0)
    /home/emichon/Documents/Scalingo/golang/src/github.com/Scalingo/cli/notifiers/list.go:57 +0x2cc
github.com/Scalingo/cli/notifiers.List(0x7ffe6ded0761, 0x2, 0x2, 0x6)
    /home/emichon/Documents/Scalingo/golang/src/github.com/Scalingo/cli/notifiers/list.go:36 +0x2fc
github.com/Scalingo/cli/cmd.glob..func103(0xc0000ef1e0)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ### To be Released
 
+* Bugfix if notifier send all events [#453](https://github.com/Scalingo/cli/pull/453)
+
 ### 1.14.1
 
 * Fix Git remote URL parsing to detect app name on Outscale [#443](https://github.com/Scalingo/cli/pull/443)

--- a/notifiers/list.go
+++ b/notifiers/list.go
@@ -16,7 +16,7 @@ func List(app string) error {
 	if err != nil {
 		return errgo.Notef(err, "fail to get Scalingo client")
 	}
-	resources, err := c.NotifiersList(app)
+	notifiers, err := c.NotifiersList(app)
 	if err != nil {
 		return errgo.Mask(err, errgo.Any)
 	}
@@ -29,11 +29,15 @@ func List(app string) error {
 		return errgo.Notef(err, "fail to list event types")
 	}
 
-	for _, r := range resources {
+	for _, notifier := range notifiers {
+		selectedEvents := "All"
+		if !notifier.GetSendAllEvents() {
+			selectedEvents = eventTypesToString(eventTypes, notifier.GetSelectedEventIDs())
+		}
 		t.Append([]string{
-			r.GetID(), string(r.GetType()), r.GetName(),
-			strconv.FormatBool(r.IsActive()), strconv.FormatBool(r.GetSendAllEvents()),
-			eventTypesToString(eventTypes, r.GetSelectedEventIDs()),
+			notifier.GetID(), string(notifier.GetType()), notifier.GetName(),
+			strconv.FormatBool(notifier.IsActive()), strconv.FormatBool(notifier.GetSendAllEvents()),
+			selectedEvents,
 		})
 	}
 	t.Render()

